src/main/java/com/conveyal/gtfs/validator/MTCValidator.java

```diff
@@ -10,7 +10,8 @@
 
 /**
  * MTCValidator runs a set of custom validation checks for GTFS feeds managed by MTC in Data Tools.
- * The checks consist of validating field lengths at this time per the 511 MTC guidelines at
+ * At this time, the checks consist of validating field lengths and that calendars apply to
+ * at least one day of the week per the 511 MTC guidelines at
```

Comment:
Oops, one mistake found ^^. Comment above needs to have (statement about calendars removed).

Reply:
Yep, just fixed.